src/bitcoin-cli.cpp

```diff
@@ -569,6 +569,56 @@ class NetinfoRequestHandler : public BaseRequestHandler
         "> bitcoin-cli -netinfo help\n"};
 };
 
+/** Process addressinfo requests */
+class AddressinfoRequestHandler : public BaseRequestHandler
+{
+private:
+    static constexpr uint8_t m_networks_size{5};
```

Comment:
57eb71ce0f27c0353ea0742a696c6d04ce048e15

Drop, and use `m_networks` size?

Reply:
I would if could omit the template arguments.